### PR TITLE
ssb-project-cli 1.0.1

### PR DESCRIPTION
**Pull request by damirmedakovic**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
-name = "hack2022-dapla-hurtigstart"
-version = "1.0.0.1"
+name = "ssb-project-cli"
+version = "1.0.1"
 description = "Hack2022 Dapla Hurtigstart"
 authors = ["Arne Sørli <81353974+arneso-ssb@users.noreply.github.com>"]
```
Comment (Andilun) [inline pyproject.toml:5]: Should be "Statistics Norway <mmw@ssb.no>"

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
-name = "hack2022-dapla-hurtigstart"
-version = "1.0.0.1"
+name = "ssb-project-cli"
+version = "1.0.1"
 description = "Hack2022 Dapla Hurtigstart"
```
Comment (mmwinther) [inline pyproject.toml:4]: This needs changing too

File: pyproject.toml (hunk 1 above)
Comment (mmwinther) [inline pyproject.toml:5]: And this

**Review by mmwinther — CHANGES_REQUESTED** [review]

**Review by Andilun — CHANGES_REQUESTED** [review]
Should be:
authors = ["Statistics Norway <mmw@ssb.no>"]

**Review by mmwinther — APPROVED** [review]
LGTM

**Review by Andilun — APPROVED** [review]

**Pull request merged**